File: IDSFintechPortal.Api/Models/ProductResponsibility.cs
namespace IDSFintechPortal.Api.Models
{
    public class ProductResponsibility
    {
        public int Id { get; set; }
        public string Responsibility { get; set; } = string.Empty;
        public string? Description { get; set; }
        public int ProductId { get; set; }
        public int TeamMemberId { get; set; }
    }
}

File: ids-fintech-portal-frontend/src/types/ProductResponsibility.ts
export interface ProductResponsibility {
    id: number;
    productId: number;
    teamMemberId: number;
    responsibility: string;
    description: string | null;
}

export interface CreateProductResponsibilityDto {
    productId: number;
    teamMemberId: number;
    responsibility: string;
    description?: string;
}